recipes/tasklib/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "tasklib" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tasklib-{{ version }}.tar.gz
+  sha256: 5ccd731b52636dd10457a8b8d858cb0d026ffaab1e3e751baf791bf803e37d7b
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - backports.zoneinfo  # [py<39]
```

Comment:
You can't use selector with `noarch` recipe but you don't have to.
The `backports.zoneinfo` conda-forge package won't install anything for python >= 3.9. It's one of those [empty packages](https://conda-forge.org/docs/maintainer/knowledge_base.html#empty-python-packages) in that case. You can put it as run requirement for all python versions.